data/roles/gecko_t_osx_1100_m1.yaml

```diff
@@ -9,7 +9,7 @@ worker_metadata:
   provisionerId: "%{lookup('gw.provisioner_id')}"
 
 worker:
-  taskcluster_version: 40.0.3
+  taskcluster_version: 44.0.0
```

Comment:
In the future, this should match what is running in production.  The current prod is 43.2.0.  I think 44.0.0 is acceptable this time around because dhouse had to build a darwin arm64 version with simple engine by hand.

https://firefox-ci-tc.services.mozilla.com/__version__

Reply:
Oh shoot - I just realized I based my patch on https://github.com/mozilla-platform-ops/ronin_puppet/pull/329 - so this isn't actually my change :). The most recent commit (https://github.com/mozilla-platform-ops/ronin_puppet/pull/341/commits/a88b9615e70151d733e5b8708558973a34bdc39f) is the only thing I added. I can rebase if you want, although I have no idea what's deployed right now.

Reply:
:zap: I'll apply this change. We apply puppet manually now on the newer macs (r8's and m1's) so I'll apply it and we can wait on #329 to go through.